webpack.config.babel.js

```diff
@@ -102,13 +102,15 @@ const PROD_CONFIG = {
   'SMS_DELIVERY_URL': 'https://telephony.onfido.com',
   'PUBLIC_PATH' : `https://assets.onfido.com/web-sdk-releases/${packageJson.version}/`,
   'RESTRICTED_XDEVICE_FEATURE_ENABLED': true,
-  WOOPRA_DOMAIN
+  WOOPRA_DOMAIN,
+  'ENV': 'production'
```

Comment:
NODE_ENV is passed below, so this doesn't seem necessary (below as well)

Reply:
Done